scripts/plugins/alerts/security/retransmissions/user_scripts/flow/retransmissions.lua

```diff
@@ -0,0 +1,78 @@
+--
+-- (C) 2019-20 ntop.org
+--
+
+local user_scripts = require("user_scripts")
+local flow_consts = require("flow_consts")
+
+-- ############################################
+
+local script = {
+	-- Script category
+	category = user_scripts.script_categories.security,
+
+	-- NOTE: hooks defined below
+	hooks = {},
+
+	gui = {
+		i18n_title = "retransmissions.retransmissions_alert_title",
+		i18n_description = "retransmissions.retransmissions_alert_description",
+	}
+}
+
+-- ############################################
+
+local function retransmissions_check(now)
+	local tcp_stats = flow.getTCPStats()
+	local c2s_retransmissions = tcp_stats["cli2srv.retransmissions"] 
+	local s2c_retransmissions = tcp_stats["srv2cli.retransmissions"]
+
+	local cli_sent_packets = flow.getPacketsSent()
```

Comment:
TCP issues were already pre-calculated in C, see `flow.getClientTCPIssues()`. Did you consider using/extending this method=

It could be too costly to perform all these operations in Lua.